scripts/reindex-images/scan-dynamo-table-for-keys.js

```diff
@@ -0,0 +1,87 @@
+const AWS = require("aws-sdk");
```

Comment:
is this the same as https://github.com/guardian/grid/pull/2771?

Reply:
Sorry, needs to be diffed from #2771, please review that or I can recreate the PR and use #2771 as base (as I should have done when creating this PR)

Reply:
Oh wait, you can make the change whilst the PR is open! C'est extra!